### DISCUSSION TITLE
Welcome to PollQ Discussions!

### DISCUSSION THREAD

**Discussion by namansharma3007** [namansharma3007/PollQ#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Karan1962** [comment]
<img width="960" alt="DarkForgotPass" src="https://github.com/user-attachments/assets/7be36a7c-4e52-40cd-9c90-15ba05ca54f5">
<img width="960" alt="DarkForgotPass2" src="https://github.com/user-attachments/assets/7bca094b-2837-4323-abb6-0fa7d0486d44">
<img width="960" alt="darkLightLogin" src="https://github.com/user-attachments/assets/f7b40eec-f9e9-4606-a5a8-65bec023311b">
<img width="960" alt="DarkLightSignup" src="https://github.com/user-attachments/assets/9934e964-805a-4633-b881-b3dbdd0861e0">
<img width="960" alt="darkLogin" src="https://github.com/user-attachments/assets/c53cabfe-8337-468b-b8ec-f92e864019ba">
<img width="960" alt="darkSignUp" src="https://github.com/user-attachments/assets/f2bc197d-49c7-4979-9cf4-18a472e9f26c">

hello @namansharma3007  shared some ui's for signup page , login page and for forgot pass page kindly suggest for the same or any changes u may want

**Comment by Karan1962** [comment]
@namansharma3007  On commit 69dc53c307861260a2805ba4bc28c338fb03ce9c  someone unnecessarily may be by mistakenly removed routes for login signup and forgot password due which those pages are not rendering should I raise a new issue for that ? need your suggestion as I am new to opensource !

**Reply by namansharma3007** [reply]
You can do it and make a commit
